src/GeolocatorService.Reactive.Uno.WinUI/GeolocatorService.Reactive.Uno.WinUI.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.44">
 	<PropertyGroup>
-		<TargetFrameworks>net6.0-windows10.0.18362;netstandard2.0;net6.0-ios;net6.0-macos;net6.0-android</TargetFrameworks>
-		<LangVersion>10.0</LangVersion>
+		<TargetFrameworks>net8.0;net8.0-ios;net8.0-android;net8.0-windows10.0.22621.0;net8.0-macos</TargetFrameworks>
+		<LangVersion>13.0</LangVersion>
```

Comment:
Should it not be 12?
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/language-versioning#defaults